README.md

```diff
@@ -1,24 +1,173 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
 
-Things you may want to cover:
+  <h3><b>Recipe App</b></h3>
 
-* Ruby version
+</div>
 
-* System dependencies
+<!-- TABLE OF CONTENTS -->
 
-* Configuration
+# 📗 Table of Contents
 
-* Database creation
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Database initialization
+# 📖 [Recipe-app] <a name="about-project"></a>
 
-* How to run the test suite
+> Recipe App is a web application that allows users to create and share recipes with other users. Users can create an account, log in, and create, edit, and delete their recipes. Users can also view recipes created by other users.
 
-* Services (job queues, cache servers, search engines, etc.)
+**[Recipe-app]**
 
-* Deployment instructions
+## 🛠 Built With <a name="built-with"></a>
 
-* ...
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Client</summary>
+<ul>
+<li>N/A</li>
+ </ul>
+<summary>Server</summary>
+<ul>
+<li>Ruby</li>
+<li>Ruby on Rails</li>
+</ul>
+<summary>Database</summary>
+<ul>
+<li>PostgreSQL</li>
+</ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+<ul>
+<li>Gitflow execution</li>
+<li>Login page</li>
+<li>Signup page</li>
+<li>Use of models</li>
+<li>Add food</li>
+<li>Add recipe</li>
+<li>View recipes</li>
+<li>View foods</li>
+<li>Add inventory</li>
+<li>View inventory</li>
+
+
+</ul>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+Coming soon.
+
+### Prerequisites
+
+In order to run this project you need:
+
+ <ul>
+<li>Node.js</li>
+<li>npm</li>
+<li>ruby 2.7.0</li>
+<li>rails   
+</li>
+<li>PostgreSQL</li>
+ </ul>
+
+### Setup
+
+Clone this repository to your desired folder:
+Use Terminal:
+
+cd my-folder
```

Comment:
[OPTIONAL]
- I would suggest placing the code that you want the user to enter in code blocks. It looks more professional that way and it is easier to read :+1: